test/runtests.jl

```diff
@@ -52,5 +75,56 @@ datadir = joinpath(@__DIR__,"data")
     @test isequal(por, [0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8])
     @test isequal(lit, [1,2,3,4,5,6,7,8])
     @test isequal(sat, [0.8,0.7,0.8,0.7,0.8,0.7,0.8,NaN])
+
+    # test if storing/leading recovers data
+    fname = tempname()*".gslib"
+    GslibIO.save_legacy(fname, sdata)
+    sdatarestored = GslibIO.load_legacy(fname, (2,2,2))
+    porrestored = sdatarestored[:Porosity]
+    litrestored = sdatarestored[:Lithology]
+    satrestored = sdatarestored[Symbol("Water Saturation")]
+    @test isequal(por, porrestored)
+    @test isequal(lit, litrestored)
+    @test isequal(sat, satrestored)
```

Comment:
```suggestion
    ndata = GslibIO.load_legacy(fname, (2,2,2))
    @test sdata == ndata
```

Reply:
It seems there is a problem with `sdata == ndata`:
Look at the output when I run the code manually:
```julia
julia> ndata = GslibIO.load_legacy(fname, (2,2,2))
2×2×2 RegularGrid{Float64,3}
  variables
    └─Lithology (Float64)
    └─Porosity (Float64)
    └─Water Saturation (Float64)

julia> sdata
2×2×2 RegularGrid{Float64,3}
  variables
    └─Lithology (Float64)
    └─Porosity (Float64)
    └─Water Saturation (Float64)

julia> sdata == ndata
false
```

I don't see any difference and comparing the internal data separately,  they are equal.